cabot/celeryconfig.py

```diff
@@ -27,4 +28,27 @@
     },
 }
 
+CELERY_QUEUES = (
+    Queue('checks'),
+    Queue('updates'),
+)
+
+CELERY_ROUTES = {
+    'run-all-checks': {
+        'queue': 'celery'
+    },
+    'update-service': {
+        'queue': 'updates'
+    },
+    'update-instance': {
+        'queue': 'updates'
+    },
+    'update-shifts': {
+        'queue': 'updates'
+    },
+    'clean-db': {
+        'queue': 'updates'
```

Comment:
I put update-shifts and clean-db in the same queue as update-service because they run super infrequently (once every 30 min and once every day), so it didn't seem worth it to have a queue dedicated to them